CHANGELOG.md

```diff
@@ -51,6 +51,7 @@ our [guidelines for writing a good changelog entry](https://github.com/biomejs/b
 - Add [noHeadImportInDocument](https://biomejs.dev/linter/rules/no-head-import-in-document/). Contributed by @kaioduarte
 - Add [noImgElement](https://biomejs.dev/linter/rules/no-img-element/). Contributed by @kaioduarte
 - Add [guardForIn](https://biomejs.dev/linter/rules/guard-for-in/). Contributed by @fireairforce
+- Add [noUselessStringRaw](https://github.com/biomejs/biome/pull/4263). Contributed by @fireairforce
```

Comment:
it seems i have added the changelog here, what else i need to add ? @Conaclos

Reply:
Sorry I missed it. All is perfect :)